explainer.md

```diff
@@ -1,624 +1,397 @@
-elements and associated properties (transform, clip, effects inherited by
-descendents) when creating pseudo elements. This is a consideration for future
-iterations of the feature.
-
-## Container/Child Split
-
-One consideration is to render each shared element using a replaced element
-directly instead of creating a container element. The motivation behind this
-split is to provide a stacking context to cross-fade the content of old and new
-shared elements. This is necessary to ensure blending identical pixels is a
-no-op using
-[plus-lighter](https://drafts.fxtf.org/compositing/#porterduffcompositingoperators_plus_lighter)
-blending. While same-origin transitions could work around this, it enables
-future extensibility for cross-origin transitions where cross-fading identical
-images would be common.
-
-## Natively Supporting Animating Box Decoration CSS Properties
-
-An alternate approach to the setup described in
-[Animating Box Decoration CSS Properties](#live-animatable-properties) is to
-support this natively in the browser by introducing a new `content-element()`
-function. This function would behave similarly to the `element()` function
-except skipping the following properties when painting the element: box
-decorations and visual effects which generate a stacking context. The image
-would also be sized to the element's content-box (as opposed to the border-box
-used by the element() function). The motivation for supporting this natively
-would be to make these properties animatable instead of requiring developers to
-implement it themselves.
+# Cross-fading
+
+Cross-fading two DOM elements is currently impossible if both layers feature transparency. This is due to the default composition operation: black with 50% opacity layered over black with 50% opacity becomes black with 75% opacity.
+
+However, the [plus-lighter](https://drafts.fxtf.org/compositing/#porterduffcompositingoperators_plus_lighter) compositing operation does the right thing when isolated to a set of elements whose `opacity` values add to 1. The "transition part" wrapper performs this isolation.
+
+Allowing `mix-blend-mode` to be set to `plus-lighter` will enable developers to create real cross-fades between elements for this feature and elsewhere.
+
+# Relation to `element()`
+
+CSS has an [`element()`](https://drafts.csswg.org/css-images-4/#element-notation) feature which allows the appearance of an element to be used as an image.
+
+This doesn't quite match either of the cases where we need to capture an element as a texture.
+
+When capturing 'as a single texture', it seems much easier for developers if we expand the capture to include things outside the border box, such as box shadows. `element()` clips at the border box.
+
+When capturing 'as a container + child texture', we capture the combination of the element's children as texture, clipped to the content box. Again this is different to `element()`.
```

Comment:
For this also, add it in the "computed style + content" section? I like the idea of introducing the concept and then referring existing concepts which come close to what we want.